Documentation/cassandra.md

```diff
@@ -86,6 +86,12 @@ To get a cqlsh shell in your new Cluster:
 The operator supports scale up of a rack as well as addition of new racks. To make the changes, you can use `kubectl edit clusters.cassandra.rook.io rook-cassandra`:
  * To scale up a rack, change the `Spec.Members` field of the rack to the desired value.
  * To add a new rack, append the `racks` list with a new rack. Remember to choose a different rack name for the new rack.
+ 
+## Scale Down
```

Comment:
i missed this in the first PR, but this doc should have a header table so that it gets included in the left-hand navigation table of contents on the generated docs site: https://rook.io/docs/rook/v0.8/quickstart-toc.html

You should add Cassandra to that page, and also add a header to the top of this `cassandra.md` page, similar to the following:

https://github.com/rook/rook/blob/master/Documentation/cockroachdb.md is a good example:
```
---
title: Cassandra
weight: 7
indent: true
---
```

Reply:
Alright, will do! 
I was wondering what those were for, I though it was some weird property of markdown I hadn't heard of :sweat_smile: